cli/cmd/project/edit.go

```diff
@@ -1,29 +1,54 @@
 package project
 
 import (
-	"fmt"
+	"context"
 
+	"github.com/rilldata/rill/admin/client"
+	"github.com/rilldata/rill/cli/cmd/cmdutil"
 	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
 	"github.com/spf13/cobra"
 )
 
 func EditCmd(cfg *config.Config) *cobra.Command {
-	var name, displayName, prodBranch string
+	var name, description, prodBranch string
 	var public bool
 
 	editCmd := &cobra.Command{
 		Use:   "edit",
 		Args:  cobra.ExactArgs(1),
 		Short: "Edit",
-		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("not implemented")
+		RunE: func(cmd *cobra.Command, args []string) error {
+			sp := cmdutil.Spinner("Updating project...")
+			sp.Start()
+
+			client, err := client.New(cfg.AdminURL, cfg.AdminToken())
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			// Todo how will get the org name? will it be flag with cmd.
```

Comment:
It should use the default org

Reply:
yeah, added this comment when getDefault org was not there :)